.gitlab-ci.yml

```diff
@@ -69,7 +69,7 @@ variables:
   RELEASE_VERSION_7: nightly
   DATADOG_AGENT_BUILDIMAGES: v1709713-2adac2a
   DATADOG_AGENT_BUILDERS: v1793959-4e4c346
-  DATADOG_AGENT_WINBUILDIMAGES: v1876043-a759bdd
+  DATADOG_AGENT_WINBUILDIMAGES: v2021523-34e07eb
```

Comment:
this is the image that's built with https://github.com/DataDog/datadog-agent-builders/pull/121 right?

Reply:
Yes (you can check, the commit hash is the second part of the image name)